### PR TITLE
build: adds portfolio fixes

### PR DESCRIPTION
**Pull request by matthew-garrett**
- Lightning bolt change
- Add divider line above Total row in Portfolio Holdings table
- Total Earnings in Total row should reflect NOTE incentives as well as organic earnings
- Updates width of portfolio side nav notification dot

### CODE REVIEW

File: packages/features/portfolio/src/components/total-earnings-tooltip/total-earnings-tooltip.tsx [hunk 1]
```diff
@@ -20,27 +23,65 @@ interface FirstValueProps {
...
+              <FirstValue
+                theme={theme}
+                isNegative={nonNoteEarnings?.includes('-')}
+              >
+                {nonNoteEarnings}
+              </FirstValue>
+            </ValueContainer>
+          </Box>
+        ) : (
+          <>
```
Comment (jeffywu) [inline packages/features/portfolio/src/components/total-earnings-tooltip/total-earnings-tooltip.tsx:69]: For consistency can we use a Box here as well

File: packages/features/portfolio/src/containers/portfolio-holdings/use-detailed-holdings.tsx [hunk 2]
```diff
@@ -189,6 +204,8 @@ export function useDetailedHoldings() {
       }
     );
 
+    console.log({ totals });
```
Comment (jeffywu) [inline packages/features/portfolio/src/containers/portfolio-holdings/use-detailed-holdings.tsx:207]: Remove the log

File: packages/features/portfolio/src/components/total-earnings-tooltip/total-earnings-tooltip.tsx (hunk 1 above)
Comment (jeffywu) [reply packages/features/portfolio/src/components/total-earnings-tooltip/total-earnings-tooltip.tsx:69]: What is the difference between the first part of the if statement and the second?

Comment (matthew-garrett) [reply packages/features/portfolio/src/components/total-earnings-tooltip/total-earnings-tooltip.tsx:69]: The first part is only for the total row and shows note usd and combined organic usd values. The second part of the conditional shows four values: note, note usd, token value and the token value usd.

**Pull request merged**